src/graphql/types/scalar/on-chain-address.ts

```diff
@@ -17,7 +17,7 @@ const OnChainAddress = new GT.Scalar({
 
 function validOnChainAddressValue(value) {
   // TODO: verify/improve. Use bc1/tb1 prefixes?
-  if (value.match(/^[A-Fa-f0-9]+$/i)) {
+  if (value.match(/^[A-Z0-9]+$/i)) {
```

Comment:
did you test this? maybe we should have a helper with some unit tests

Reply:
We cannot constrain btc addresses to only contain uppercase. Lower case addresses are not only valid but the norm for most wallets.

Reply:
This the current check we have. It's case insensitive.

Reply:
Ah right `/i` -> case insensitive.